src/test/java/com/wikia/webdriver/elements/mercury/components/discussions/common/Reply.java

```diff
@@ -76,9 +74,17 @@ public void waitForVoteCountChangeTimeLagToPass() {
   }
 
   public boolean hasImage() {
-    changeImplicitWait(3, TimeUnit.SECONDS);
+    setShortImplicitWait();
```

Comment:
can't  you avoid changing implicit wait explicitly on that level in this case?

Reply:
I can, but in cases where I want to check negative scenario (`assertFalse(Reply.hasImage())`) I'd rather not wait 15 seconds when I'm sure it won't show up